test/integration/test_main.py

```diff
@@ -6,51 +6,163 @@
 
 # First-party
 from pytrajplot import main
-
-
-hres_test_args = [
-    ['1_altitudes', '1_altitudes', {'datatype': 'png', 'domain': 'europe'}],
-    ['2_altitudes', '2_altitudes', {'datatype': 'png', 'domain': 'europe'}],
-    ['3_altitudes', '3_altitudes', {'datatype': 'png', 'domain': 'europe'}],
-    ['4_altitudes', '5_altitudes', {'datatype': 'png', 'domain': 'europe'}],
+from pathlib import Path
+
+# Arguments and expected outputs for pytrajplot tests
+test_sets = [
+
+    # HRES tests
+    {
+        'model': 'hres',
+        'arguments': ['1_altitudes', '1_altitudes', {'datatype': 'png', 'domain': 'europe'}],
+        'expected_output': [
+            './test_hres/1_altitudes/20210503T12_Linate_LAGRANTO-IFS-HRES-Europe_Trajektorien_F_048_europe.png'
```

Comment:
From what I can tell, only the name of the expected_output is used, so we can remove all the relative paths before the filenames
```suggestion
            '20210503T12_Linate_LAGRANTO-IFS-HRES-Europe_Trajektorien_F_048_europe.png'
```